### PR TITLE
HPCC-32110-1 Fixing issue with sec3, sect2, programlisting in PT_BR docs

### PR DESCRIPTION
**Pull request by Michael-Gardner**
## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Test Build: https://github.com/Michael-Gardner/HPCC-Platform/actions/runs/9651366413/job/26619059050

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@JamesDeFabia I think you're right about the diff we handed off to the Portuguese translation team being too large. This section that includes YAML_Filestruct-Placement needs to be looked at.  It likely completely duplicates the section it had an id conflict with.  But I don't speak Portuguese and I don't know how Greg wants to have the final book look like, so I made some minor changes and got rid of a smaller ENV duplicate section so the tags would line up properly for a build.

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV. I recognized enough to confirm that is was duplicated text.

File: docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml [hunk 1]
```diff
@@ -1034,7 +1034,7 @@
         <para><ulink
         url="https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md">https://github.com/hpcc-systems/HPCC-Platform/blob/master/helm/hpcc/docs/placements.md</ulink></para>
 
-        <sect4 id="YAML_FileStruct_Placement">
+        <sect4 id="YAML_FileStruct-Placement">
```
Comment (ghalliday) [inline docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1037]: I think this should be sect3, and line 1098 should be /sect3.  
If it builds I won't worry though..

Comment (Michael-Gardner) [reply docs/PT_BR/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml:1037]: It builds.  I'm rather certain that entire <sect4 id="YAML_FileStruct-Placement"> ... </sect4> is a duplicate that should be removed. But I'm not certain exactly what parts of it should be removed, which one is the correct one, etc.  Greg made a lot of changes to that specific file and I think the people doing the translation were confused when they saw the diff.

**Review by ghalliday — COMMENTED** [review]
Approving so that 9.6.x can be built.

**Comment by ghalliday** [comment]
@g-pan I suspect this still isn't the correct fix because the section levels do not nest correctly.

**Pull request merged**